flink-runtime/src/main/java/org/apache/flink/runtime/io/network/api/CheckpointBarrier.java

```diff
@@ -119,6 +119,6 @@ public boolean isCheckpoint() {
 	}
 
 	public CheckpointBarrier asUnaligned() {
-		return checkpointOptions.isUnalignedCheckpoint() ? this : new CheckpointBarrier(getId(), getTimestamp(), getCheckpointOptions().asTimedOut());
+		return checkpointOptions.isUnalignedCheckpoint() ? this : new CheckpointBarrier(getId(), getTimestamp(), getCheckpointOptions().toUnaligned());
```

Comment:
part of a previous fixup?

Reply:
No, I think it belongs to the right commit. With `CheckpointOptions` hotfix/refactoring, this method converts it to `unaligned`, which can't be timed out.